### ISSUE TITLE
download problem

### ISSUE THREAD

**Issue by ake020675** [tools-dataset/Industrial-Tools-Detection-Dataset#1]
Great work! This dataset should be very helpful.
 However it's quit inconvenient to download from BaiduNetDisk.
Would u like to ZIP all files to one single file?

**Comment by CPor99** [comment]
> 
> 
> Great work! This dataset should be very helpful.
> However it's quit inconvenient to download from BaiduNetDisk.
> Would u like to ZIP all files to one single file?

Hi @ake020675, did you manage to download the dataset? Thanks :)

**Comment by mona0809** [comment]
I'm also unable to download the dataset since I cannot create a Baidu account from outside of China :(

**Comment by andreasoprani** [comment]
I have the same problem, it would be awesome to have this dataset available to the rest of the world too.

**Comment by ake020675** [comment]
> > Great work! This dataset should be very helpful.
> > However it's quit inconvenient to download from BaiduNetDisk.
> > Would u like to ZIP all files to one single file?
> 
> Hi @ake020675, did you manage to download the dataset? Thanks :)

No. As i said ,there are too many images and i need to download them 1 by 1.

**Comment by luiscastro1995** [comment]
I would like to use the Dataset for my thesis but I am having problems downloading it.
In the [Baidu link](https://pan.baidu.com/s/12BRdIFVPh-xDZ6-ZW9bHeA#list/path=%2Fsharelink1579703523-167233640360977%2FITD%20Dataset&parentPath=%2Fsharelink1579703523-167233640360977) I tried to download the folders, however a .deb file is downloaded instead. I also tried to install that .deb file but it installed the BaiduNetDisk application. To open that application I need to create an account but was not able to create one because I need a Chinese phone number.
Is there a simple solution like a zip file or any alternative like the kind?
I only need the RGB and Depth images if it makes any difference.

Thank you!

**Comment by MustafaAskar** [comment]
hello, did anyone manage to download this dataset from baidu?
